.travis.yml

```diff
@@ -1,10 +1,12 @@
 language: ruby
 sudo: false
 cache: bundler
+install:
+  bundle install
```

Comment:
Hd to do this explicitly after the container infrastructure blog post that travis guys sent out.